corehq/privileges.py

```diff
@@ -87,7 +87,7 @@
     DATA_CLEANUP,
     TEMPLATED_INTENTS,
     CUSTOM_INTENTS,
-    # BUILD_PROFILES,
+    BUILD_PROFILES,
```

Comment:
the assumption i make in my code is that all privileges are added to max privileges (roles - max_privileges == all plans). it wouldn't make sense to add daily saved exports to the build profile privilege. also talked to cal and this is supposed to be in max privs